config/rbac/role.yaml

```diff
@@ -15,8 +15,17 @@ rules:
   - delete
   - get
   - list
+  - patch
   - update
   - watch
+- apiGroups:
+  - '*'
+  resources:
+  - secrets/finalizers
+  verbs:
+  - get
+  - patch
+  - update
```

Comment:
Why do we need new apiGroups?

Reply:
My previous change adds 'finalizers' to the objects we create (secrets, quoteattestation) to protect from unnecessary deletion from outside of the TCS. But, I missed changing the RBAC configuration.

Reply:
https://github.com/intel/trusted-certificate-issuer/pull/19

Reply:
I think @poussa means that why it has to be `'*'` and not some particular api groups, such as `""` for secrets and `tcs.intel.com` for quoteattestation.

Reply:
I am not sure, this is auto-generated, let me check.

Reply:
I fixed the kubebuilder markup to `""` , instead of `"*"`.

Reply:
I fixed the kubebuilder markup to used "" as apiGroup instead of "*" for Secrets.